docs/source/spec/core.rst

```diff
@@ -4045,6 +4045,112 @@ settings from a service.
             }
         }
 
+The values for ``outputToken`` and ``items`` are paths. Paths are a series of
+identifiers separated by dots (``.``) where each identifier represents a
+member name in a structure. The first member name MUST correspond to a member
+of the output structure and each subsequent member name MUST correspond to a
+member in the previously referenced structure. Paths MUST adhere to the
+following ABNF.
+
+.. productionlist:: smithy
+    path    :`identifier` *("." `identifier`)
```

Comment:
Why limit this to two levels deep?

Reply:
It isn't limited to two levels deep - you'll get a warning if it's more than that but your build will still pass. The warning exists because every level of nesting makes the api progressively more cumbersome to use. In an ideal world we would only have the result list and the output token in the response - anything else makes it hard to provide useful abstractions like iterators. A single level of nesting can be justified if it's a result wrapper shape, which is a fairly common pattern.

Reply:
Ok, that sounds reasonable.